Makefile

```diff
@@ -80,3 +80,5 @@ genericinstall:
 	make && \
 	sudo make install
 
+shellcheck:
+	shellcheck `grep -ril "/bin/bash" *|grep -Ev "yml|yaml"`
```

Comment:
maybe make the grep more accepting with `grep -Erl '#! ?/bin/b?a?sh'`
and to avoid the yaml special case you can make that
```bash
grep -Erl '^#! ?/bin/b?a?sh'
```

Reply:
Ah that a pretty good point and much nicer, let me change it!